src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -204,6 +217,15 @@ export default class AircraftController {
      */
     addItem = (item) => this.aircraft.list.push(item);
 
+    /**
+     * Get the instance of AircraftCommander that was created by this Controller
+     *
+     * @returns {AircraftCommander}
+     */
+    getAircraftCommander() {
+        return this._aircraftCommander;
+    }
```

Comment:
This is a good opportunity for a _getter_:
```javascript
get aircraftCommander() {
    return this._aircraftCommander;
}
```

which is accessed just like a property, ie `this._aircraftController.aircraftCommander`. Any time you are simply accessing something that you don't want to be reassigned, using a getter like this is the way to go.

Reply:
done